report/src/main/webapp/app/layout/main/url/reports/screen_layout.html

```diff
@@ -29,6 +29,9 @@
       </label>
     </div>
   </div>
+  <div class="try-new button button-small button-blue">
```

Comment:
Could you please use more descriptive class name here?
E.g. `advanced-screen-comparison` or smth like that

Reply:
Done